vite.config.js

```diff
@@ -2,55 +2,77 @@ import { dirname, resolve } from 'path';
 import { fileURLToPath } from 'url';
 
 // vite.config.js
-import { defineConfig } from 'vite';
+import { defineConfig, loadEnv } from 'vite';
 import vue from '@vitejs/plugin-vue';
 import vuetify from 'vite-plugin-vuetify';
 
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = dirname(__filename);
 
 // https://vitejs.dev/config/
-export default defineConfig({
-  plugins: [
-    vue(),
-    vuetify({
-      autoImport: false,
-      styles: { configFile: 'src/sass/variables.scss'}
-    }),
-  ],
-  resolve: {
-    alias: {
-      _: resolve(__dirname),
-      '@': resolve(__dirname, './src'),
-    },
-  },
-  server: {
-    port: 8080,
-    strictPort: true,
-    proxy: {
-      '/game': {
-        target: 'http://localhost:1337',
-        changeOrigin: true,
-      },
-      // Required for the health response to work on the client
-      '/health': {
-        target: 'http://localhost:1337',
-        changeOrigin: true,
+export default defineConfig(({ mode }) => {
+  // Load env file based on `mode` in the current working directory.
+  // Set the third parameter to '' to load all env regardless of the `VITE_` prefix.
+  // See https://vitejs.dev/config/#using-environment-variables-in-config
+  const env = loadEnv(mode, process.cwd(), '');
+  let HOST_SERVER_URL = 'http://localhost:1337';
+  if (env.CUTTLE_DOCKERIZED === 'true') {
```

Comment:
If we're dockerized, we need to change a few values so the client proxies to the server properly.